bowtie/templates/report.html.j2

```diff
@@ -190,6 +190,10 @@
                   </th>
                   <td>
                     <small class="font-monospace text-muted">{{ implementation.get("version", "") }}</small>
+
+                    <button type="button" class="btn btn-sm btn-secondary" data-bs-toggle="modal" data-bs-target="#implementation-{{ loop.index }}-runtime-info">
+                      more info
```

Comment:
Let's maybe make this even more conspicuous with one of those [i icons](https://icons.getbootstrap.com/icons/info-circle-fill/). Make sense?

Reply:
Sure!